roles/heat/tasks/main.yml

```diff
@@ -54,6 +54,17 @@
     - heat-engine
   when: os == 'rhel' and openstack_install_method == 'source'
 
+- name: install heat services (rhel source)
```

Comment:
So doing it this way we will have duplicate openstack-heat-<service> files. The package install will drop them in /usr/lib/systemd/system/openstack-heat*, and this task will put unit files in /etc/systemd/system/openstack-heat* . So if we wanted to use this approach we'd have to have a different name for the services so name = heat-api instead of name = openstack-heat-api . 

I think we should avoid re-writing the systemd unit files if we can. In this case, it looks like the package puts this in it's systemd unit files:

ExecStart=/usr/bin/heat-api --config-file /usr/share/heat/heat-dist.conf --config-file /etc/heat/heat.conf

So Whatever we put into /etc/heat/heat.conf will override what's in dist.conf. 

I've started heat api and looked in the logs as to what values are being loaded, and it looks like we might just need to add this to our /etc/heat/heat.conf

[paste_deploy]
api_paste_config = /etc/heat/api-paste.ini

api_paste_config seemed to be the only value that was being pulled in from the dist-conf file, as we did not have this set in our heat.conf.